### DISCUSSION TITLE
Report an issue for AdGurd Home Version: v0.105.2

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#2967] [General]
AdGurd Home Version: v0.105.2                                                                                  Report an issue

--> Filters --> DNS blocklists --> Ad blocklist --> Choose from list --> TUR: nurcan Turk ad-list

Error: control/filtering/add_url | Couldn't fetch filter from url https://raw.githubusercontent.com/xorcan/hosts/master/xhosts.txt: got status code != 200: 404 | 400

**Comment by ameshkov** [comment]
Yeah, we'll remove that list
